frontends/web/src/components/forms/checkbox.module.css

```diff
@@ -69,19 +69,19 @@
 .success input:checked + label::before {
     background-color: var(--color-success);
     border-color: var(--color-success);
-    color: var(--color-white);
+    color: var(--color-alt);
 }
 
 .warning input:checked + label::before {
     background-color: var(--color-warning);
     border-color: var(--color-warning);
-    color: var(--color-white);
+    color: var(--color-alt);
 }
 
 .info input:checked + label::before {
     background-color: var(--color-info);
     border-color: var(--color-info);
-    color: var(--color-white);
+    color: var(--color-alt);
 }
 
```

Comment:
and some more at the end of this file

```
checkbox input[disabled] + label::before {
    background: rgb(248, 248, 248);
    border-color: rgb(220, 220, 220);
```

Not sure keep as is or update?

Reply:
Like the comment above, I kept these hard coded values because we don't currently have those colors as variables. I think we can keep it for now because I think the check boxes don't need to be changed for dark mode. And then we can make them variables in a future pr.